src/tools/nav_export/obj_exporter.h

```diff
@@ -53,7 +53,7 @@ class ObjExporter
 
     printf( "[Obj] Finished exporting %s in %lu ms\n",
             fileName.substr( fileName.find( "pcb_export" ) - 1 ).c_str(),
-            std::chrono::duration_cast< std::chrono::milliseconds >( end - start ).count() );
+            static_cast<unsigned long>(std::chrono::duration_cast< std::chrono::milliseconds >( end - start ).count() ));
```

Comment:
this static_cast is not ideal
prefer changing the format specifier %lu to match .count() type, for example %ld